test_howdoi.py

```diff
@@ -166,7 +167,24 @@ def test_get_questions(self):
         expected_output = ['https://stackoverflow.com/questions/40108569/how-to-get-the-last-line-of-a-file-using-cat-command']
         actual_output = howdoi._get_questions(links)
         self.assertSequenceEqual(actual_output, expected_output)
-
+    
+    def test_help_queries(self):
+        help_queries =  self.help_queries
+
+        for query in help_queries:
+            output = self.call_howdoi(query)
+            self.assertTrue(output)
+            assert 'retrieve n number of answers' in output
```

Comment:
I think `self.assertTrue` might be better than simply `assert`, because the former is part of the `unittest` framework.